### PR TITLE
Issue #141138 feat: TJQueue Component

### PR DESCRIPTION
**Pull request by ahghatol**

### CODE REVIEW

File: src/com_tjqueue/media/tjqueueconsume.php [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+	/**
+	 * Method to acknowledge that message processed successfully and can be deleted from queue
+	 *
+	 * @return  object Message
+	 *
+	 * @since 1.0
+	 */
+	public function receive()
+	{
+		$this->message = $this->consumer->receive(20000);
```
Comment (ahghatol) [inline src/com_tjqueue/media/tjqueueconsume.php]: @ahghatol  Check what is 20000 here

**Comment by ahghatol** [comment]
@coolbung  / @manojLondhe  please review PR

**Comment by ahghatol** [comment]
@coolbung / @manojLondhe 
Note:  Queue list view is not implemented as of now so you can ignore the code written for that.

File: src/com_tjqueue/media/helpers/tjqueuecontext.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
+<?php
```
Comment (coolbung) [inline src/com_tjqueue/media/helpers/tjqueuecontext.php]: Move this as a class in libraries folder in the component admin folder

File: src/core/consumers/email.php [hunk 3]
```diff
@@ -0,0 +1,128 @@
+<?php
```
Comment (coolbung) [inline src/core/consumers/email.php]: Path in the repo should be src/plugins/tjqueue/core

File: src/com_tjqueue/media/helpers/tjqueuecontext.php (hunk 2 above)
Comment (ahghatol) [reply src/com_tjqueue/media/helpers/tjqueuecontext.php]: As per discussion after install tjqueue library will be installed in:  administrator/components/com_tjqueue/libraries/

Comment (ahghatol) [reply src/com_tjqueue/media/helpers/tjqueuecontext.php]: This is done

**Pull request merged**